docs/cloud/execution/k8s_job_environment.md

```diff
@@ -32,17 +32,17 @@ kind: Job
 metadata:
   name: my-prefect-job
   labels:
-    identifier: ""
-    flow_run_id: ""
+    identifier: ''
+    flow_run_id: ''
```

Comment:
@joshmeek you know more about whether this is OK in a yaml file or not; I suspect we want to revert to double quotes here

Reply:
I _think_ it's fine but I personally prefer double quotes

Reply:
I'm 98% sure it's fine from a technical perspective, but since there's a preference I'll swap back.